### PR TITLE
Enable to back to shop in admin login page

### LINKED ISSUE #23775: Enable to back to shop in admin login page
## Is your feature request related to a problem?

when you arrive on the admin page by mistake, it will be interesting to have the possibility of coming back to the site (this often happens for administrators.)

## Describe the solution you'd like

Enable to back to shop in admin login page.
Only the “back to my shop” will be changed and placed on the top left corner of the page:

image

### PR DESCRIPTION
**Pull request by PululuK**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | when you arrive on the admin page by mistake, it will be interesting to have the possibility of coming back to the site (this often happens for administrators.).
| Type?             | improvement
| Category?         | FO 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/23775
| How to test?      | See https://github.com/PrestaShop/PrestaShop/issues/23775
| Possible impacts? | N/A

image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23776)

### CODE REVIEW

**Comment by Progi1984** [comment]
UX :heavy_check_mark: on issue.

**Review by Progi1984 — DISMISSED** [review]

**Comment by marionf** [comment]
@Progi1984 

I think the issue is UX :heavy_check_mark: for the idea of the feature
Otherwise, @prestascott made a [suggestion](https://github.com/PrestaShop/PrestaShop/issues/23775#issuecomment-808091271)
@PululuK Do you think it's possible to integrate his feedback ?

I remove the UX :heavy_check_mark: as the feedback of @prestascott isn't yet integrated

I put PM :heavy_check_mark: because it seems a good idea :+1:

File: admin-dev/themes/default/template/controllers/login/content.tpl [hunk 1]
```diff
@@ -164,6 +164,7 @@
 		</div>
 		{/if}
 	</div>
+    <p class='text-center'><i class="icon-caret-left"></i> {l s='Back to' d='Admin.Global.Action'} <a href='{$homeUrl}'>{$shop_name}</a></p>
```
Comment (Julievrz) [inline admin-dev/themes/default/template/controllers/login/content.tpl]:
```suggestion
    <p class='text-center'><i class="icon-caret-left"></i> {l s='Back to' d='Admin.Actions'} <a href='{$homeUrl}'>{$shop_name}</a></p>
```

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by sarahdib** [comment]
Hello @PululuK 
What is the behavior expected in Multistore context ? Only display the first shop or all of them ? 
Thank you

**Comment by PululuK** [comment]
> Hello @PululuK
> What is the behavior expected in Multistore context ? Only display the first shop or all of them ?
> Thank you

Hi @sarahdib 

We refer to the current domain name, consider this case : 

- You have two shops (multi-shop)
- Shop 1 : www.url-shop1.com
- Shop 2 : www.url-shop2.com

By default for both stores the admin folder is the same, so : 

- www.url-shop1.com/adminblabla will redirect to www.url-shop1.com
- www.url-shop2.com/adminblabla will redirect to www.url-shop2.com

**Comment by Robin-Fischer-PS** [comment]
Hi @PululuK !

It's OK on desktop and mobile display.

With Multistore, only the default shop URL is displayed, since the admin URL is the same for all the shops. It's not the optimal behavior, but it's OK, as seen with @marionf .

So it's QA :heavy_check_mark: 

pr 23776 desktop display

**Comment by Progi1984** [comment]
Thanks @PululuK & @Robin-Fischer-PS

**Comment by Robin-Fischer-PS** [comment]
@Progi1984 @PululuK 

I just realized that [the suggested design ](https://github.com/PrestaShop/PrestaShop/issues/23775#issuecomment-808091271)by @prestascott was not applied. Would it be possible to revert the merge, in order to change the design ?

Sorry for that.

**Comment by PululuK** [comment]
Hi @Robin-Fischer-PS @Progi1984 

I can open another PR in this case.

**Comment by Progi1984** [comment]
Perfect @PululuK :)

**Comment by Robin-Fischer-PS** [comment]
Thanks @PululuK , and sorry again, I was too quick to shoot the QA :heavy_check_mark: :yum:

**Pull request merged**